### PR TITLE
chore: Unlock dependencies for main package and bundle size action

### PR DESCRIPTION
**Pull request by pan-kot**
The unlock dependencies action needs to take into account the workflows/bundle-size package-lock as well.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/actions/unlock-dependencies/index.js [hunk 1]
```diff
@@ -8,22 +8,44 @@ const path = require("path");
...
+    });
+
+    Object.keys(packageLock.dependencies).forEach((dependencyName) => {
+      removeDependencies(dependencyName, packageLock.dependencies);
+    });
+
+    fs.writeFileSync(filename, JSON.stringify(packageLock, null, 2) + "\n");
+
+    console.log(
+      "Removed @cloudscape-design/ dependencies from package-lock file"
```
Comment (gethinwebster) [inline .github/actions/unlock-dependencies/index.js:37]: minor: might be useful to include `filename` here for debugging

**Review by gethinwebster — APPROVED** [review]

**Comment by just-boris** [comment]
This workflow runs for all repositories, not only for components. 

I am afraid that running it globally may cause big impact issues.

Can we find another fix which does not put us at risk of exploding everything?

**Review by connorlanigan — APPROVED** [review]

**Comment by pan-kot** [comment]
> This workflow runs for all repositories, not only for components.
> 
> I am afraid that running it globally may cause big impact issues.
> 
> Can we find another fix which does not put us at risk of exploding everything?

@just-boris, what kind of issues do you imagine? I am planning to also update the script in components to remove the cloudscape dependencies from the bundle-size package-lock.

**Comment by just-boris** [comment]
Following the principle of least surprises, I would put a separate unlock script for directly into bundle size workflow.

Workflows should be more or less standalone, this dependency on a script from a global workflow is not obvious

**Comment by pan-kot** [comment]
Alright, I'll do that for now

**Pull request closed without merging**